plugins/parsers/prometheusremotewrite/README.md

```diff
@@ -42,3 +42,32 @@ prompb.WriteRequest{
 ```
 prometheus_remote_write,instance=localhost:9090,job=prometheus,quantile=0.99 go_gc_duration_seconds=4.63 1614889298859000000
 ```
+
+**For alignment with the [InfluxDB Prometheus Remote Write Spec v1.8](https://docs.influxdata.com/influxdb/v1.8/supported_protocols/prometheus/#how-prometheus-metrics-are-parsed-in-influxdb)**
+
+- Use starlark to rename the measurement name to the fieldname and rename the fieldname to value. Use namepass to only apply the script to prometheus_remote_write metrics
```

Comment:
Maybe write "Use the [Starlark processor](https://github.com/influxdata/telegraf/tree/master/plugins/processors/starlark) to rename the measurement name to the fieldname and rename the fieldname to value. Use `namepass` to only apply the script to prometheus_remote_write metrics"